RELEASE_NOTES

```diff
@@ -1,3 +1,5 @@
+- Bug Fix: FENIX-15402 - CgdIntegrationService: NumberFormatException when student number not a number. Ticket ISCTE-FENIXEDU-637
```

Comment:
@ajsco Release notes is not in the standar format in this case since you have added both tickets I suggest to do:
```
- Bug Fix: CgdIntegrationService: NumberFormatException when student number not a number. (FENIX-15402) [ISCTE-FENIXEDU-637]
```

Reply:
Done